src/weaponJP.js

```diff
@@ -0,0 +1,28 @@
+(function(){
+
+// Check page correctness
+if (!location.hash.match("#gacha/weapon")) {
+  const answer = confirm("You don't seem to be in #gacha/weapon page. Run anyway?");
+  if (answer === false) return;
+}
+
```

Comment:
Might be a good idea to have a check if locale is (not) jp and warn the user here